src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -177,6 +178,9 @@ public WebAssemblyHost Build()
             // Intentionally overwrite configuration with the one we're creating.
             Services.AddSingleton<IConfiguration>(Configuration);
 
+            // Add configuration for logging builder
+            Logging.AddConfiguration(Configuration);
```

Comment:
Looks like this causes three extra assemblies to survive the linker based on the diff above. If we also add an `appsettings.json` file, that makes 4 extra HTTP requests but doesn't change the default app behavior. If this reasoning is correct, I'm not 100% sold that this is a desired tradeoff 😄 

Would it work instead for the developer to call `builder.Logging.AddConfiguration(builder.Configuration)` if they specifically want this?